enterprise/reporting/error_extractor.go

```diff
@@ -353,39 +350,31 @@ func getErrorCodeFromStatTags(statTags map[string]string) string {
 }
 
 func (ext *ExtractorHandle) isVersionDeprecationError(errorMessage string) bool {
```

Comment:
are there any tests?

Reply:
yes all passing

Reply:
we don't need to maintain original behavior as this only for internal monitoring purpose.